MIGRATION.md

```diff
@@ -2,8 +2,15 @@
 
 The guides below are provided to ease the transition of existing applications using the Onfido SDK from one version to another that introduces breaking API changes.
 
-## `5.6.0` -> [Next version]
-With release [Next version] there are breaking changes that will affect integrators with customised languages or UI copy.
+## `5.6.0` -> [next-version]
```

Comment:
We should update with the release version here since the string change below is part of release 5.10.0
I think it should also read as `5.6.0` -> `5.10.0`. I must have missed updating this part of the doc when I did the merge PR.

Reply:
Updated